caim_base/migrations/utils/set_values_for_non_nulls.py

```diff
@@ -0,0 +1,31 @@
+
+AGE_TEMP_DEFAULT = 0
```

Comment:
Could these functions not be put directly in the migration file? I've got a constitutional fear of `utils` directories

Reply:
I don't keep functions like this outside migration file for one stupid reason, if I need to import them into the shell or for tests I need to be able to import them. I can't really import these functions easily from a migration file because in Django migrations file name always start with numbers. I know that sounds dumb, but try it and you'll see python files that start with numbers breaks some import module syntax, it's possible to get around, but it's not really worth the hassle. It's probably smart to split these up though.